types/google-apps-script/google-apps-script-tests.ts

```diff
@@ -25,3 +25,9 @@ function createAndSendDocument() {
   // Send yourself an email with a link to the document.
   GmailApp.sendEmail(email, subject, body);
 }
+
+// Regression
```

Comment:
Ah, I meant the bugs I wrote in the PR body:

> - `Service.Restriction` was incorrectly named `Service`.
> - Enum fields had wrong types.

Should I write comments more precisely?